### PR TITLE
Ncco objects

### PR DESCRIPTION
**Pull request by judy2k**
Implementing NCCO objects before implementing server-side code to handle webhook requests.

### CODE REVIEW

**Comment by judy2k** [comment]
Dammit, I missed that eventUrl is always a list of URLs.

File: src/main/java/com/nexmo/client/voice/ncco/InputNcco.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    }
+
+    @Override
+    public String getAction() {
+        return ACTION;
+    }
+
+    @Override
+    public String toJson() {
+        return NccoSerializer.getInstance().serializeNcco(this);
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/voice/ncco/InputNcco.java]: Smart. I like this

File: src/main/java/com/nexmo/client/voice/ncco/ConnectNcco.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+    public Integer getTimeout() {
+        return timeout;
+    }
+
+    public void setTimeout(Integer timeout) {
+        this.timeout = timeout;
+    }
+
+    public Integer getLimit() {
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/voice/ncco/ConnectNcco.java]: Will the API sometimes not return this field?

File: src/main/java/com/nexmo/client/voice/ncco/ConnectNcco.java [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+    public String getFrom() {
+        return from;
+    }
+
+    public void setFrom(String from) {
+        this.from = from;
+    }
+
+    public Integer getTimeout() {
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/voice/ncco/ConnectNcco.java]: Will the API sometimes not return this field?

**Pull request merged**